### PR TITLE
UWA-6 Added Carpentry services list and tested the painting services …

### PR DESCRIPTION
**Pull request by satyanagkumar**
Added the Carpentry list Services and also the filter by location in the painting page, please review my code if there are any errors.

### CODE REVIEW

**Comment by rohitparapati** [comment]
Is this code working for a distinct location in painting service page?

**Comment by satyanagkumar** [comment]
> Is this code working for a distinct location in painting service page?

Yes first it displays services of all locations when user selects the different location it shows only the services limited to that location

File: src/controller.js
Comment (rohitparapati) [inline src/controller.js:1]: this code will display the list of services in painting and cleaning services and the use of filter location in painting services list.

**Review by rohitparapati — APPROVED** [review]
good work.

**Pull request merged**